install.sh

```diff
@@ -105,7 +105,7 @@ adjust_bin() {
 	"$root/$prefix/libexec/$bin"
     cat > "$root/$prefix/bin/$bin" <<EOF
 #!/bin/bash -e
-export GNUTLS_SYSTEM_PRIORITY_FILE="\${GNUTLS_SYSTEM_PRIORITY_FILE-$prefix/libreloc/gnutls.config}"
+export GNUTLS_SYSTEM_PRIORITY_FILE="\${GNUTLS_SYSTEM_PRIORITY_FILE:-$prefix/libreloc/gnutls.config}"
 export LD_LIBRARY_PATH="$prefix/libreloc"
 exec -a "\$0" "$prefix/libexec/$bin" "\$@"
```

Comment:
What does this fix?

Reply:
A syntax error.  It was supposed to provide a default value for the parameter substitution.
```
man bash(1):

       ${parameter:-word}
              Use Default Values.  If parameter is unset or null, the expansion of word is substituted.  Otherwise, the value of parameter is substituted.
       ${parameter:=word}
```

Reply:
I'll send it as an independent patch...
It's an orthogonal fix

Reply:
Works for me:

```sh
$ x=aa
$ echo ${x-bb}
aa
$ echo ${y-bb}
bb
```

The bash manual has this hidden piece of extra info:

```
   When not performing substring expansion, using the form described
below (e.g., ':-'), Bash tests for a parameter that is unset or null.
Omitting the colon results in a test only for a parameter that is unset.
Put another way, if the colon is included, the operator tests for both
PARAMETER's existence and that its value is not null; if the colon is
omitted, the operator tests only for existence.
```

I used the colon-less form so that if the user explicitly set the variable to null, it is not changed. Only if the variable does not exist I want it to be set.

Reply:
I wasn't aware of that feature... thanks

Reply:
Well, I had to search the manual for it, because I never remember all the variations.